thinkhazard/static/less/report.less

```diff
@@ -149,6 +149,26 @@ body {
   top: 10px;
   left: 10px;
   z-index: 2;
+
+  height: 42px;
+  margin: 5px;
+  background: rgba(255,255,255, 0.5);
+  border-radius: 5px;
```

Comment:
It looks like all the button have the same border radius.
Let's set it globally by overriding bootstrap's variables.